docs/pages/docs/guides/continuous-integration.mdx

```diff
@@ -107,3 +107,74 @@ jobs:
             fetch-depth: 2
       # ...
 ```
+
+## CircleCI
+
+The following example shows how to use Turborepo with CircleCI and yarn.
+
+Start by creating a `.circleci/config.yml` file
+
+```yaml
+version: 2.1
+orbs:
+  node: circleci/node@5.0.2
+workflows:
+  test:
+    jobs:
+      - test
+jobs:
+  test:
+    docker:
+      - image: cimg/node:lts
+    # To use Remote Caching, uncomment the next lines and follow the steps below.
+    # environment:
+    #  TURBO_TOKEN: $TURBO_TOKEN
+    #  TURBO_TEAM: $TURBO_TEAM
+    steps:
+      - checkout
+      - node/install-packages:
+          pkg-manager: yarn
+      - run:
+          command: yarn turbo run test
+```
+
+#### Remote Caching
```

Comment:
we might want to figure out a way to condense this section with the other remote caching one. It doesn't look/read too bad right now since its just GitHub Actions and CircleCI, but I imagine once we add a third thing itll get awkward

Reply:
We now have a `<Tab>` construct that we can use since a person will only ever need one of them.

Added here: https://github.com/vercel/turborepo/pull/1072